source/documentation/other-topics/custom-default-backend.md

```diff
@@ -0,0 +1,78 @@
+
+#### Setup
+
+Custom default-backend can be managed inside a namespace by creating a custom default-backend service and deployment of the custom error pages container in your namespace. Configure the name of the service resource created that Nginx should use as the default backend for requests that don’t match any configured ingress rules (hostname and path combinations) using the annotation in your Ingress.
+
+##### Create custom error page
+First create a docker image containing custom HTTP error pages using the [example][ingress-nginx-custom-error-pages] from the ingress-nginx, or [simplified version][cloud-platform-custom-error-pages] of that created by the cloud platform team.
+
+##### Customised default backend
+Using this [custom-default-backend][customized-default-backend] example from ingress-nginx create a service and deployment of the error pages container in your namespace.
+
+```
+$ kubectl -n ${namespace} create -f custom-default-backend.yaml
+service "nginx-errors" created
+deployment.apps "nginx-errors" created
+```
+
+This should have created a Deployment and a Service with the name nginx-errors.
+
+```
+$ kubectl -n ${namespace} get deploy,svc
+NAME                           DESIRED   CURRENT   READY     AGE
+deployment.apps/nginx-errors   1         1         1         10s
+
+NAME                   TYPE        CLUSTER-IP  EXTERNAL-IP   PORT(S)   AGE
+service/nginx-errors   ClusterIP   10.0.0.12   <none>        80/TCP    10s
+```
+
+##### Defining annotations in Ingress file.
+
+Final step is to to use an [Default Backend][default-backend-annotation] annotation in your Ingress, This annotation is of the form nginx.ingress.kubernetes.io/default-backend: <svc name> to specify a custom default backend. This <svc name> is a reference to a service inside of the same namespace in which you are applying this annotation. This annotation overrides the global default backend.
+
+This service will handle the response when the service in the Ingress rule does not have active endpoints. If a default backend annotation is specified on the ingress, the errors will be routed to that annotation's default backend service (instead of the global default backend).
+
+Example Usage:
+```
+annotations:
+  # This is referencing the SAME NAMESPACE that this resource is in
+  nginx.ingress.kubernetes.io/default-backend: nginx-errors
+```
+
+This default-backend service also handle the error responses if both [default-backend][default-backend-annotation] annotation and the [custom-http-errors][custom-http-error-annotation] annotation is set. To configure which HTTP status codes Nginx should be forwarding to the default-backend using the `custom-http-errors` property, which can be configured using the [default-backend][default-backend-annotation] annotation in your Ingress.
```

Comment:
This section is a bit hard to understand. What would be an example scenario where the default backend service would serve the page? Or, why would the users want to set things up this way?

Reply:
Explained it with the example below